### PR TITLE
Show data for proper sector

### PR DESCRIPTION
**Pull request by gbielanski**
Please review

### CODE REVIEW

File: app/src/main/java/pl/example/android/garbageapp/data/database/SectorTermDao.java [hunk 1]
```diff
@@ -12,8 +12,8 @@
 @Dao
 public interface SectorTermDao {
 
-    @Query("SELECT * FROM sector_terms WHERE term >= :term AND sectorType == :sectorType")
-    LiveData<List<SectorTerm>> getFutureSectorTerms(Date term, int sectorType);
+    @Query("SELECT * FROM sector_terms WHERE term >= :t AND sectorType == :sectorType")
+    LiveData<List<SectorTerm>> getCurrentSectorTerms(Date t, SectorType sectorType);
```
Comment (miltomasz) [inline app/src/main/java/pl/example/android/garbageapp/data/database/SectorTermDao.java]: Change param name, 't' is not correct.

File: app/src/main/java/pl/example/android/garbageapp/data/network/SectorTermsNetworkDataSource.java [hunk 2]
```diff
@@ -98,27 +101,34 @@ public void startSectorTermsSyncService() {
...
+
+        SectorValueEventListener(SectorType sectorType){
+            super();
+            this.sectorType = sectorType;
+        }
+
         @Override
         public void onDataChange(DataSnapshot dataSnapshot) {
-            List<Sector> sectorModel = new ArrayList<>();
+            List<SectorTerm> sectorModel = new ArrayList<>();
```
Comment (miltomasz) [inline app/src/main/java/pl/example/android/garbageapp/data/network/SectorTermsNetworkDataSource.java]: Change name of the variable: sectorModel --> sectorTerms.

**Review by miltomasz — CHANGES_REQUESTED** [review]
Please correct 2 variable names.

File: app/src/main/java/pl/example/android/garbageapp/data/database/SectorTermDao.java (hunk 1 above)
Comment (gbielanski) [reply app/src/main/java/pl/example/android/garbageapp/data/database/SectorTermDao.java]: You are right. I was checking which one "term" or ":term" refers to method arguments and I left it accidentally.

File: app/src/main/java/pl/example/android/garbageapp/data/network/SectorTermsNetworkDataSource.java (hunk 2 above)
Comment (gbielanski) [reply app/src/main/java/pl/example/android/garbageapp/data/network/SectorTermsNetworkDataSource.java]: OK

**Comment by gbielanski** [comment]
Changes done and published. Please review.

**Review by miltomasz — APPROVED** [review]

**Pull request merged**